projects/ngx-wig/src/lib/ngx-wig.module.ts

```diff
@@ -0,0 +1,54 @@
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA, NO_ERRORS_SCHEMA, ModuleWithProviders, InjectionToken } from '@angular/core';
```

Comment:
`CUSTOM_ELEMENTS_SCHEMA` and `NO_ERRORS_SCHEMA` are unused

Reply:
yes, true